examples/Python3/demo4.py

```diff
@@ -0,0 +1,69 @@
+import snowboydecoder
+import sys
+import signal
+import speech_recognition as sr
```

Comment:
Please write in the document how you can install this dependency.

Reply:
Done. I'm happy to add more information if necessary.